src/abseil-cpp/preprocessed_builds.yaml.gen.py

```diff
@@ -166,7 +166,16 @@ def resolve_hdrs(files):
 
 
 def resolve_srcs(files):
-  return [ABSEIL_PATH + "/" + f for f in files if f.endswith(".cc")]
+  srcs = [ABSEIL_PATH + "/" + f for f in files if f.endswith(".cc")]
+  # Replace absl/random/internal/randen_hwaes.cc with
+  #         src/abseil-cpp/randen_hwaes_nohw.c
```

Comment:
did you write `randen_hwaes_nohw.c` by hand or how did you obtain it?
Since what you're doing seems quite dirty, I think there needs to be a more detailed description of what you've done in a comment. Right now I kind of know what was the point but I wouldn't be able to do it myself since important information seems to be missing (so maintaining this hack in the future will be difficult).

Reply:
Yes, you're right. I wrote this `randen_hwaes_nohw.cc` which is meant to forward all operations to the non-hw one. My plan is to use this one-off hack for a few month (proving this is a working solution) and work with the Abseil team to have an option to disable HW acceleration completely. I guess we'll be able to replace it with the formal one from the next LTS version of Abseil.

Reply:
Your plan sounds fine but a detailed description of the workaround and the plan needs to be provided in a comment.
Also please make sure the src/abseil-cpp/randen_hwaes_nohw.cc has an introductory comment that mentions the context or links to a place where context can be found.

Reply:
Sure! I'll do.

Reply:
Done! PTAL